src/Appwrite/Utopia/Response/Model/Build.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class Build extends Model
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('$id', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Build ID.',
+                'default' => '',
+                'example' => '5e5ea5c16897e',
+            ])
+            ->addRule('startTime', [
+                'type' => self::TYPE_INTEGER,
+                'description' => 'The deployment creation date in Unix timestamp.',
+                'default' => 0,
+                'example' => 1592981250,
+            ])
+            ->addRule('deploymentId', [
+                'type' => self::TYPE_STRING,
+                'description' => 'The deployment that created this build',
+                'default' => '',
+                'example' => '5e5ea5c16897e',
+            ])
+            // Build Status
```

Comment:
If we need this docblock for maintainers, we should definitely have it for devs. `description` is automatically used in the generated model docs.